src/scenes/CreateEvent/store.js

```diff
@@ -59,7 +59,7 @@ const messages = defineMessages({
 
 const MAX_LEN_EVENTNAME_HEX = 640;
 const MAX_LEN_RESULT_HEX = 64;
-const TIME_DELAY_FROM_NOW_SEC = 15 * 60;
+const TIME_DELAY_FROM_NOW_SEC = 0;
```

Comment:
this shouldnt be needed anymore since we dont have a range?

Reply:
ok, removed.